mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/CassandraMessageManager.java

```diff
@@ -35,6 +35,7 @@
 import org.apache.james.mailbox.quota.QuotaManager;
```

Comment:
again a commit that could make its own PR? (f78b408cc2c6ce64aee938928c8fbd01f9a0a364)

Reply:
It requires step #13 higly coupled to the  content of this pull request... Extracting #13 represents probably a man day of work...